tests/Directory.Build.targets

```diff
@@ -26,18 +26,19 @@
   <ItemGroup>
     <!--Test Dependencies-->
     <PackageReference Update="BenchmarkDotNet" Version="0.12.1" />
-    <PackageReference Update="BenchmarkDotNet.Diagnostics.Windows" Version="0.12.1" Condition="'$(OS)' == 'Windows_NT'" />    
+    <PackageReference Update="BenchmarkDotNet.Diagnostics.Windows" Version="0.12.1" Condition="'$(OS)' == 'Windows_NT'" />
     <PackageReference Update="Colourful" Version="2.0.5" />
     <PackageReference Update="coverlet.collector" Version="1.3.1-preview.27.gdd2237a3be" PrivateAssets="All"/>
     <PackageReference Update="Magick.NET-Q16-AnyCPU" Version="7.22.0" />
-    <PackageReference Update="Microsoft.DotNet.RemoteExecutor" Version="5.0.0-beta.20069.1" />
-    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="16.5.0" />
-    <PackageReference Update="Moq" Version="4.14.5" />
+    <PackageReference Update="Microsoft.DotNet.RemoteExecutor" Version="6.0.0-beta.20513.1" />
+    <PackageReference Update="Microsoft.DotNet.XUnitExtensions" Version="6.0.0-beta.20513.1" />
```

Comment:
Are we only adding this because of `ActiveIssueAttribute`?

Reply:
Ahh, I see it's not a trivial feature because of the optional `TargetFrameworkMonikers` attribute. Cool 👍

Reply:
Yeah, it's pretty nifty. Some cool other attributes available also.

Reply:
Not actually needed now but keeping it as it's far too useful!